streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -196,15 +195,15 @@ private void logRestoreOffsets(final TopicPartition partition, final long starti
     private Collection<TopicPartition> completed() {
         final Set<TopicPartition> completed = new HashSet<>(stateRestorers.keySet());
         completed.removeAll(needsRestoring.keySet());
-        log.debug("{} completed partitions {}", logPrefix, completed);
+        log.debug("completed partitions {}", completed);
```

Comment:
Capitalize `completed`

Reply:
`completed` is in lower case because before this we have applied a `logPrefix` using `LogContext`